raiden/storage/serialization/serializer.py

```diff
@@ -55,7 +55,7 @@ def serialize(obj: Any) -> Dict:
             try:
                 schema = SchemaCache.get_or_create_schema(obj.__class__)
                 data = schema.dump(obj)
-            except (TypeError, ValidationError) as ex:
+            except (TypeError, ValidationError, ValueError) as ex:
```

Comment:
LGTM but why the ValueError catching here?

Reply:
There was one test that failed, catching the exception here seemed like the best fix. It's related to some changes in the serialization code in marshmallow.

Reply:
Can you remember which one it was? Some times catching ValueError feels like catching `Exception` :sweat_smile:

Reply:
It was 
https://github.com/raiden-network/raiden/blob/0cc8ddf22b3c86d4739ab305eee68dd5b356ba48/raiden/tests/unit/test_serialization.py#L61-L66

Reply:
Are you sure? That test uses the `JSONSerializer` and you are catching the `ValueError` in the `DictSerializer`

Reply:
Yes, `JSONSerializer` uses `DictSerializer` internally.

Reply:
Aha okay I see. I assumed that was not the case since it was not subclassing it.

So that means that with the new marshmallow trying to do schema load() where the type does not match (in the test it should be an int and we give bytes) can also raise `ValueError`.

Incidentally we already catch `ValueError` in `deserialize()` so that should also be fine.